pom.xml

```diff
@@ -1525,6 +1525,10 @@
     <ci.rerunFailingTestsCount>0</ci.rerunFailingTestsCount>
     <!-- Limit ITs to 30 minutes by default -->
     <test.fork.timeout>1800</test.fork.timeout>
+
+    <!-- OpenAPI-->
+    <cxfServiceDescriptionOpenapiVersion>3.2.4</cxfServiceDescriptionOpenapiVersion>
```

Comment:
nit:  Place it in alphabetical order in above list. don't need to separate it out.

Reply:
Updated.